### PR TITLE
Generating Random Datasets With Trumania

### LINKED ISSUE #7022: Generating Random Datasets with Trumania
#### Proposed title of article
Generating Random Datasets with Trumania
#### Proposed article introduction
This article will instill the need as to why you might want to generate random datasets and what to expect from them. It will also describe how to use Trumania, a data generation Python library. Generating random dataset is relevant both for data engineers and data scientists.
#### Key takeaways
1.Why generate random datasets  
2. Schema-Based Random Data Generation
3. Generating Inter-related Data with Trumania
4. Creating a Trumania Circus
#### Article quality
The article is of importance as it will provide the reader with expound  understanding on how to use trumania  to generate random datasets. Every step and point will be described in detail to give the needed knowledge and understanding.
#### References
N/A

### PR DESCRIPTION
**Pull request by kymomolly**
#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #7022

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by louisefindlay23** [comment]
Plagiarism check complete

File: content/articles/generating-random-datasets-with-trumania/index.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+- Creating time parameters of the story results in a more realistic distribution of the messages throughout the given time.
+- Addition of other relationships to define a social network.
+
+We will have a better understanding in the next section.
+
+### Creating a Trumania circus
+A circus is a data generator that is used to control and monitor processes and sockets. It can be driven in the web interface, python API, and command-line interface.
+
+#### How to create a circus
+A trumania circus is created using the simple steps as follows:
```
Comment (Dawe-7) [inline content/articles/generating-random-datasets-with-trumania/index.md:94]:
```suggestion
A trumania circus is created using the code below:
```

**Review by Dawe-7 — CHANGES_REQUESTED** [review]
Awesome article @kymomolly. I've left you a few suggestions that i think will improve the quality of your article.

Ping me when you have effected the changes.

File: content/articles/generating-random-datasets-with-trumania/index.md (hunk 1 above)
Comment (kymomolly) [reply content/articles/generating-random-datasets-with-trumania/index.md:94]: working on it

**Comment by kymomolly** [comment]
Hello @Dawe-7, I have incorporated the changes you requested.  Please have a look at the article. Thanks

**Review by Dawe-7 — APPROVED** [review]
Nice article @kymomolly. Kindly go through the changes made throughout the review process so as to improve on the flow and format of your future articles.

**Pull request merged**